reaper-adm-extension/src/reaper_adm/CMakeLists.txt

```diff
@@ -196,6 +197,7 @@ find_package(WDL REQUIRED QUIET)
 
 target_include_directories(reaper_adm_object
     PRIVATE
+	${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I couldn't figure out where this is needed so just checking it's not a leftover

Reply:
It's for locating AppConfig.h. Now that update_check.cpp is in a different directory, that header is not within the search paths. Which AppConfig.h to include depends on which target its being used in, so it should be in the search paths rather than using a long path to it.

Reply:
Actually, since update_check was split in update_check  and update_check_settings_file, we can move update_check back to the extension directory and avoid this - it's function is bespoke to the extension anyway